### PR TITLE
feat: help post management

### PR DESCRIPTION
**Pull request by avasilic**
Signed-off-by: net-tech <hello@nettech.link>

### CODE REVIEW

File: src/types/Types.ts [hunk 1]
```diff
@@ -6,4 +11,12 @@ import { Comment, LikeUser, Notification, Profile } from "./DatabaseSchemas"
 
 type ReturnData = Comment[] | Notification[] | LikeUser[] | Profile[] | Object
 
-export { ReturnData }
+/**
+ * Interactions that can be replied or edited to.
+ */
+type ReplyableInteraction =
```
Comment (tm21cy) [inline src/types/Types.ts]: Repliable, not Replyable

File: src/utilities/general.ts [hunk 2]
```diff
@@ -54,6 +55,23 @@ class Util {
...
+	/**
+	 * Replies to an interaction or edits the reply if the interaction has already been replied to. Sets ephemeral to undefined if the interaction has already been replied to.
+	 * @param {RepliableInteraction} interaction The interaction to reply to.
+	 * @param {InteractionReplyOptions} content The content to reply with.
+	 */
+	public static replyOrEdit(
+		interaction: RepliableInteraction,
+		content: InteractionReplyOptions
+	) {
+		if (interaction.replied) {
```
Comment (tm21cy) [inline src/utilities/general.ts:68]: Ternary statement?

**Review by tm21cy — CHANGES_REQUESTED** [review]
Review above

File: src/utilities/general.ts (hunk 2 above)
Comment (avasilic) [reply src/utilities/general.ts:68]: No, because you can't do `content.ephemeral = undefined` in a clean way in a ternary.

**Review by tm21cy — APPROVED** [review]

**Pull request merged**